server/uploader.js

```diff
@@ -204,11 +251,26 @@ uploader.bigWigToBCWig = function(prefix, bwFile, bigWigToWigAddr, uploadPath) {
       var fd = fs.openSync(segFile, 'w');
       fs.writeSync(fd, segBuf, 0, offset, 0);
       fs.closeSync(fd);
+      percentage += 4 / chrNum;
```

Comment:
This part seems to be exactly the same as above. Consider creating a nested function.